backend/crud.py

```
@@ -314,3 +315,184 @@ def delete_spot(db: Session, spot_id: UUID, user_id: UUID) -> None:
+
+
+def get_quest_by_id(db: Session, quest_id: UUID) -> Optional[models.Quest]:
+    """IDでクエストを取得"""
+    return _quest_query(db).filter(models.Quest.id == quest_id).first()
+
+
+def create_quest(db: Session, requester_id: UUID, payload: schemas.QuestCreate) -> models.Quest:
+    """新規クエスト作成"""
+    requester = get_user_by_id(db, requester_id)
+    if requester is None:
+        raise ValueError(f"requester {requester_id} not found")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)